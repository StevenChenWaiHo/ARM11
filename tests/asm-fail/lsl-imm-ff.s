lsl r1,#0xff
