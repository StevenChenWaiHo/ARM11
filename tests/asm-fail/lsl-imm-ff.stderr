tests/asm-fail/lsl-imm-ff.s:1:13: Const too large for a shift const: 255 (max is 31)
